lms/static/sass/base/_variables.scss

```diff
@@ -82,6 +82,27 @@ $gray-d2: shade($gray,40%); // #4c4c4c
 $gray-d3: shade($gray,60%); // #323232
 $gray-d4: shade($gray,80%); // #191919
 
+$cms-blue: rgb(0, 159, 230); // NOTE - eventually, this set of blues should be synced with CMS.  
```

Comment:
It does but the LMS blues have been adjusted to be more accessible - the base LMS blue is at the line for AA accessibility on light gray. Overall, I agree with your plan for cleanup and refactoring of the blues to be Studio-like, but I'm not sure this PR is the place for it.